src/dmd/root/outbuffer.d

```diff
@@ -396,14 +396,27 @@ struct OutBuffer
         return (cast(const char*)data)[0 .. offset];
     }
 
-    /***********************************
+    /**
      * Extract the data as a slice and take ownership of it.
+     *
+     * When `true` is passed as an argument, this function behaves
+     * like `dmd.utils.toDString(thisbuffer.extractChars())`.
+     *
+     * Params:
+     *   nullT = When `true`, the data will be `null` terminated.
+     *           This is useful to integrated with e.g. C API calls.
```

Comment:
"This is useful to **be** integrated..."

Reply:
I think it should be "This is useful to integrate~~d~~ with e.g. C API calls."

Reply:
I realized this comment was a bit broken, so I've completely rephrased it.